src/Dfe.Spi.Common/Dfe.Spi.Common.Caching/Definitions/Caches/IMemoryCacheProvider{TCacheKey,TCacheValue}.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Dfe.Spi.Common.Caching.Definitions.Caches
+{
+    /// <summary>
+    /// Describes the operations of the memory cache provider.
+    /// </summary>
+    /// <typeparam name="TCacheKey">
+    /// The type of key used in the underlying storage.
+    /// </typeparam>
+    /// <typeparam name="TCacheValue">
+    /// The type of item to store in the cache.
+    /// </typeparam>
+    public interface IMemoryCacheProvider<TCacheKey, TCacheValue>
```

Comment:
What do you foresee as the use for this interface?

Reply:
Code updated to use strings and objects, as suggested.